<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Book</class>
 <widget class="QDialog" name="Book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>311</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>630</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>35</y>
     <width>132</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>图书管理</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bookId">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>84</y>
     <width>251</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>84</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书籍号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>127</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>170</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bookName">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>127</y>
     <width>251</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="author">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>170</y>
     <width>251</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="press">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>213</y>
     <width>251</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>213</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>256</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书籍类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="kind">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>256</y>
     <width>127</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>古典文学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>信息技术</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>美食杂志</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>现代散文</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>科幻小说</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>学术论文</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>299</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>馆藏量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>342</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版日期</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="publishTime">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>342</y>
     <width>251</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="quantity">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>299</y>
     <width>89</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
